src/Microsoft.Health.Dicom.Core/Features/Delete/DeleteHandler.cs

```diff
@@ -61,16 +61,16 @@ public async Task<DeleteResourcesResponse> Handle(DeleteResourcesRequest request
 
         private static void ValidateDeleteResourcesRequest(DeleteResourcesRequest request)
         {
-            UidValidator.Validate(request.StudyInstanceUid, nameof(request.StudyInstanceUid));
+            DicomUidValidation.Validate(request.StudyInstanceUid, nameof(request.StudyInstanceUid));
```

Comment:
We had removed the Dicom prefix before. Do we need it?

Reply:
Yeah, Brandon mentioned similar things before -- we should remove prefix `Dicom`. But today I still see some inconsistency -- e.g: `DicomElementMinimumValidator`.

 I don't have good idea on this. Check the classes under Validation namespace: 
![image](https://user-images.githubusercontent.com/67345894/128725705-b137642f-1259-4ba3-9c62-fb107c833764.png)
If `UidValidator` is used, should `DicomDateValidation` to be `DateValidation` ? And if `DateValidation` is too generic?